main.go

```diff
@@ -1618,7 +1616,10 @@ func handleGetRepo(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusOK)
 		w.Write(resp)
 	} else {
-		w.WriteHeader(http.StatusInternalServerError)
+		s := fmt.Sprintf("Could not find Repo For User: %v", err)
+		log.Printf("Error found: %v", err)
+		http.Error(w, s, http.StatusBadRequest)
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
Why isn't this using the error API for 404?

Reply:
I think a lot of these instances where http.Error is still being used should probably be wraperror instead, right?

Reply:
The WriteHeader isnt needed since http.Error will do that for you so that is a good thing to point out Griffin. However http.StatusBadRequest is a wrapper but for 400 rather than 404 which is http.StatusNotFound.

Reply:
seems like this should be a 404